### PR TITLE
Fixes #8997: Remove code specific to CFEngine 3.5

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/8997

### CODE REVIEW

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
We may want to wait for a new ncf branch to do these changes.

**Comment by amousset** [comment]
Commit modified

File: tree/10_ncf_internals/configuration.cf [hunk 1]
```diff
@@ -67,9 +59,13 @@ bundle agent configuration
       "pass2" expression => "pass1";
       "pass1" expression => "any";
 
-  methods:
-    # We cannot use the generic method to expand template, as it rely on the logger,
-    # which is not yet defined
-    pass2.!cfengine_3_5::
-      "expand" usebundle => ${generic_method_expansion};
+  # We cannot use the generic method to expand template, as it rely on the logger,
```
Comment (peckpeck) [inline tree/10_ncf_internals/configuration.cf]: relies

**Comment by ncharles** [comment]
Appart the typo, this looks great ! Thank you

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
@ncharles Rebased.

**Pull request merged**